share/spice/health_scores/health_scores.js

```diff
@@ -0,0 +1,71 @@
+(function(env) {
+    "use strict";
+
+    // A Swiftype relevance value to eliminate weak,
+    // irrelevant results. This value does a good
+    // job of filtering out hits which don't match for
+    // [business name|category] [location].
+    var RELEVANCE_THRESHOLD = 0.06;
+
+    
+    env.ddg_spice_health_scores = function(api_result) {
+
+        if (!api_result || api_result.length === 0) {
```

Comment:
Looks like `api_result` gets an object and not an array so we shouldn't check for `length`:

![screen shot 2015-04-08 at 2 18 37 am](https://cloud.githubusercontent.com/assets/81969/7039900/a93d669a-dd95-11e4-87f6-72c7f7b4ff90.png)